example/httpclient/main.go

```diff
@@ -0,0 +1,32 @@
+// httpclient is an example of how to use the configured http.Client
+// that can be returned by the gh.HTTPClient function.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/cli/go-gh"
+	"github.com/google/go-github/v42/github"
```

Comment:
Let's not add this example because it adds a whole new library to our `go.mod` and it's entirely unnecessary. Go developers typically don't need an example of what to do with a `http.Client` instance.

Reply:
👍